src/CSharp/CodeCracker/Style/TaskNameAsyncAnalyzer.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+
+namespace CodeCracker.Style
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class TaskNameAsyncAnalyzer : DiagnosticAnalyzer
+    {
+        public const string DiagnosticId = "CC0061";
+        internal const string Title = "Async method can be terminating with 'Async' name.";
+        internal const string MessageFormat = "Change method name including 'Async'.";
```

Comment:
Why not `Change method name to FooAsync`, using a `MessageFormat` like `Change method name to {0}`?